### PR TITLE
Presale wallet

### PR DESCRIPTION
**Pull request by debris**
- fixed crash when presale wallet password was incorrect
- ethstore command for importing presale wallet
- parity command for importing presale wallet (geth compatible)

### CODE REVIEW

File: ethstore/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Usage:
     ethstore change-pwd <address> <old-pwd> <new-pwd> [--dir DIR]
     ethstore list [--dir DIR]
     ethstore import [--src DIR] [--dir DIR]
+    ethstore import-wallet <path> <password> [--dir DIR]
```
Comment (gavofyork) [inline ethstore/README.md]: bit horrible putting a password in the CLI invocation. better to have an interactive password prompt

**Comment by gavofyork** [comment]
lgtm assuming a positive answer above.

**Comment by debris** [comment]
parity uses password file for backwards compatibility with geth. 
ethstore uses plain text. I find it much more flexible than using files. If people want to use password files they can always do `ethstore import-wallet ethwallet.json $(cat pass.txt)`

**Comment by tomusdrw** [comment]
The problem is that by default people won't do that and their password will be readable in `$ history`.

**Comment by debris** [comment]
imho `ethstore` is a developer tool. Developers should know their environment and that typing plain password might be insecure

**Comment by gavofyork** [comment]
developers don't generally need to import a presale wallet.

people who bought in the presale do.

**Comment by gavofyork** [comment]
(if you really like, you could try password-file name as a password itself if the file doesn't exist)

**Pull request merged**